apps/drupal-default/particle_theme/templates/block/block--utc-hero--full.html.twig

```diff
@@ -37,18 +37,19 @@
 	{% endif %}
 	{# Sets colors for hero content depending on selection #}
 	{% if utc_hero.hero_color == 'Dark Blue' %}
-		{% set title_color = 'lg:text-white' %}
-		{% set body_color = 'lg:text-white' %}
-		{% set tag_color = 'lg:text-white' %}
-		{% set btn_border_color = "lg:border-white" %}
+		{% set title_color = 'text-white' %}
+		{% set body_color = 'text-white' %}
+		{% set tag_color = 'text-white' %}
+		{% set btn_border_color = "border-white" %}
 		{% set bg_color = 'background-color:#112e51' %}
 		{% set bg1_opacity = '' %}
     	{% set bg2_opacity = 'opacity-90' %}
 	{% else %}
 		{% set title_color = 'text-utc-new-blue-500' %}
 		{% set body_color = 'text-base' %}
 		{% set tag_color = 'text-base' %}
-		{% set btn_border_color = "lg:border-utc-new-blue-500" %}
+		{% set btn_border_color = "border-utc-new-blue-500" %}
+		{% set btn_class = "light-gray-hero" %}
 		{% set bg_color = 'background-color:#104dd4' %}
```

Comment:
should this use a utc theme variable instead of the color hex?

Reply:
This gray is not a theme variable. When I worked with Brittany on making the light gray useful, we had to use an off-theme color in order for the image to render correctly. Using theme variables didn't work in this case.